CITATION.bib

```diff
@@ -1,10 +1,17 @@
 @article{pymc2023,
-  title={PyMC: A Modern and Comprehensive Probabilistic Programming Framework in Python},
-  author={Abril-Pla Oriol and Andreani Virgile and Carroll Colin and Dong Larry and Fonnesbeck Christopher J. and Kochurov Maxim and Kumar Ravin and Lao Jupeng and Luhmann Christian C. and Martin Osvaldo A. and Osthege Michael and Vieira Ricardo and Wiecki Thomas and Zinkov Robert},
-  journal = {{PeerJ} Computer Science},
-  publisher = {{PeerJ}},
-  volume={9},
-  pages={e1516},
-  year={2023},
-  doi={10.7717/peerj-cs.1516}
+	title = {{PyMC}: a modern, and comprehensive probabilistic programming framework in {Python}},
```

Comment:
What about the upper case?

Reply:
First of all, the published title is not in title case: https://peerj.com/articles/cs-1516/

Besides, upper or lower case are irrelevant in bibtex entries, unless wrapped in extra `{}`s. BibTeX will apply title case or normal case to all titles depending on the style sheet.

This version preserves the original case better, by wrapping `PyMC` and `Python` in extra `{}`s (see ref. 111 of https://arxiv.org/pdf/2408.09024 likely generated from the previous bib file).